packages/zowe-explorer/CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Fixed an issue where a migrated data set is unusable after it is recalled through Zowe Explorer. [#3294](https://github.com/zowe/zowe-explorer-vscode/issues/3294)
 - Fixed an issue where a recalled PDS is expandable after it is migrated through Zowe Explorer. [#3294](https://github.com/zowe/zowe-explorer-vscode/issues/3294)
 - Fixed an issue where data set nodes did not update if migrated or recalled outside of Zowe Explorer. [#3294](https://github.com/zowe/zowe-explorer-vscode/issues/3294)
+- Fixed issue where Search operation did not prompt for credentials if profile contains expired token. [#2259](https://github.com/zowe/zowe-explorer-vscode/issues/2259)
+- Fixed issue where inactive status was not displayed for profiles loaded from Global Config. [#3134](https://github.com/zowe/zowe-explorer-vscode/issues/3134)
```

Comment:
Suggestion: We'll probably want to move these entries to the bottom of this section as the existing entries are sorted chronologically

Reply:
I kind of like to sort things chronologically as well. However, I can see some benefit of adding them in somewhat of a random order. That way you can prevent merge conflicts 😋

Reply:
Will address this in my other V2 port PR 😋